src/components/ContinuationIn/SummaryBusinessHomeJurisdiction.vue

```diff
@@ -215,7 +247,8 @@
               class="pt-4 pt-sm-0"
             >
               <div id="authorization-date">
-                {{ authorizationDate || '[Unknown]' }}
+                {{ yyyyMmDdToPacificDate(getContinuationAuthorization?.date, true, false) ||
+                  '[Unknown]' }}
```

Comment:
Do you think I should move this into a computed (getter)?

Reply:
might be a good idea to move but okay to keep like this as well

Reply:
I moved it.

Reply:
I'm all for good ideas, so I moved it.